swift/ql/src/queries/Security/CWE-916/InsufficientHashIterations.qhelp

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+  <overview>
+    <p>Using hash functions with less than 1,000 iterations is not secure. That scheme is vulnerable to password cracking attacks due to having an insufficient level of computational effort.</p>
```

Comment:
This query seems as if it's fairly closely related to the JavaScript query [Use of password hash with insufficient computational effort](https://codeql.github.com/codeql-query-help/javascript/js-insufficient-password-hash/).

I think that it might be worth reusing some of the explanation of why this is important.

Reply:
Agreed. I've updated the help text here with the relevant information from the JavaScript query. Thanks for the pointer to that query @felicitymay !